docs/src/ActivationCard.doc.js

```diff
@@ -0,0 +1,162 @@
+// @flow strict
+import React, { type Node } from 'react';
+import PropTable from './components/PropTable.js';
+import Example from './components/Example.js';
+import PageHeader from './components/PageHeader.js';
+
+const cards: Array<Node> = [];
+const card = c => cards.push(c);
+
+card(
+  <PageHeader
+    name="ActivationCard"
+    description="Activation cards are used in groups to communicate a user’s stage in a series of steps toward an overall action."
+  />
+);
+
+card(
+  <Example
+    name="Not started and Pending Cards"
+    defaultCode={`
+<Box display="flex" marginStart={-1} marginEnd={-1}>
+  <Box paddingX={1}>
+    <ActivationCard
+      status="notStarted"
+      statusMessage="Not started"
+      title="Claim your website"
+      message="Grow distribution and track Pins linked to your website"
+      link={{href: "https://pinterest.com", label:"Claim your website now"}}
+      dismissButton={{
+        accessibilityLabel: 'Dismiss card',
+        onDismiss: ()=>{},
+      }}
+    />
+  </Box>
+  <Box paddingX={1}>
+    <ActivationCard
+      status="pending"
+      statusMessage="Pending"
+      title="Claim your website"
+      message="We will notify you via email as soon as your site has been successfully claimed."
+      link={{href: "https://pinterest.com", label:"Learn more"}}
+      dismissButton={{
+        accessibilityLabel: 'Dismiss card',
+        onDismiss: ()=>{},
+      }}
+    />
+  </Box>
+</Box>
+  `}
+  />
+);
+
+card(
+  <Example
+    name="Needs attention and Complete Cards"
+    defaultCode={`
+<Box display="flex" marginStart={-1} marginEnd={-1}>
+  <Box paddingX={1} width="100%">
```

Comment:
@norrise120 could we remove the `width="100%"`? Would be good to keep this consistent across examples.